src/containers/Header/UserInfo/index.jsx

```diff
@@ -144,10 +144,10 @@ class UserInfo extends PureComponent {
   _renderUser() {
     const classes = classnames(styles.UserInfo_container_user_img, testClass('user-picture'));
     return <AdultPicture className={classes}
-        src={this.props.user.profile.avatar_url}
-        gender={this.props.user.profile.gender}
-        userFirstName={this.props.user.profile.given_name}
-        userLastName={this.props.user.profile.family_name}
+        src={this.props.user.get('profile').avatar_url}
+        gender={this.props.user.get('profile').gender}
+        userFirstName={this.props.user.get('profile').given_name}
+        userLastName={this.props.user.get('profile').family_name}
```

Comment:
Well you can still use `.get()` on a record, so we should just use that everywhere. Records are OK.